base/applications/mspaint/winproc.cpp

```diff
@@ -452,7 +486,7 @@ LRESULT CMainWindow::OnCommand(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL& bH
             break;
         }
         case IDM_HELPHELPTOPICS:
-            HtmlHelp(m_hWnd, _T("help\\Paint.chm"), 0, 0);
+            DoHtmlHelpW(m_hWnd, L"help\\mspaint.chm", HH_DISPLAY_TOPIC, 0);
```

Comment:
It looks like vulnerable.

Reply:
... due to relative path.

Reply:
It would indeed use the path relative to the so-called current drive and directory.
On the other hand, calc uses for example `"%systemroot%\\Help\\calc.chm"`
Now, if you believe that relying on %systemroot% is vulnerable, well, this means that someone already bricked your system since that system environment variable was modified.
But then, I suppose that the person didn't need to use mspaint and its htmlhelp file to do an attack...